rfi_file_monitor/utils/utils.py

```diff
@@ -0,0 +1,63 @@
+import gi
```

Comment:
Not a fan of `utils.utils` to be honest... How about just renaming this file to `__init.py`?

Reply:
been changed